lib/mayaHydra/hydraExtensions/adapters/meshAdapter.cpp

```diff
@@ -201,9 +250,25 @@ class MayaHydraMeshAdapter : public MayaHydraShapeAdapter
                 return {};
             }
             return GetPoints(mesh);
-        } else if (key == MayaHydraAdapterTokens->st) {
+        } 
+
+        if (key == HdTokens->normals) {
+            MStatus status;
```

Comment:
I am doing a change in this file to standardize the methods and avoid passing sometimes a MfnMesh and sometimes not.